client/client.go

```diff
@@ -214,8 +214,9 @@ func WithSkipStoreLimit() RegionsOption {
 
 // GetRegionOp represents available options when getting regions.
 type GetRegionOp struct {
-	needBuckets         bool
-	allowFollowerHandle bool
+	needBuckets                  bool
+	allowFollowerHandle          bool
+	outputMustContainAllKeyRange bool
```

Comment:
Do we need it?

Reply:
Yes?

It has been input in L1179

Reply:
Oh, I was mislead by batchScanRegionsOptions.